articles/azure-monitor/visualize/visualize-use-grafana-dashboards.md

```diff
@@ -163,6 +163,39 @@ You can export a dashboard as an ARM template that contains the JSON for the das
 1. In the dashboard screen, select **Export** then **Export as ARM template**.
 1. Select **Download** and save the file.
 
+## Add diagnostic settings
+
+You can configure your dashboard with diagnostic settings in order to have a record of when it was updated to a new version or restored to a previous one.
+
+You can create up to five different diagnostic settings to send the logs to independent destinations.
+
+1. In the dashboard screen, select **Diagnostic settings**, under **Monitoring**
+
+   :::image type="content" source="media/visualizations-grafana/diagnostic-settings-menu.png" alt-text="Screenshot of the Azure platform. Diagnostic settings.":::
+
+1. Select **+ Add diagnostic setting**.
+
+1. For **Diagnostic setting name**, enter a unique name.
+
+1. Select **allLogs** from the following options:
+   - **audit** streams all audit logs (Update events are audit logs, and they are currently the only logs available, so this is the same as choosing all logs.)
```

Comment:
I would change this list into an indented table like how you have the fields below - would make it easier to scan!

```suggestion
   - **audit** streams all audit logs (Update events are audit logs, and they are currently the only logs available, so this is the same as choosing all logs.)
   | Logs categories | Description |
   | ----------------- | ------------- |
   | **audit**           | Streams all audit logs. Selecting this option is the same as choosing _allLogs_, since _Update Events_ are audit logs, and are currently the only logs available. 
```